src/main/java/ua/volosiuk/mytokenservice/controller/TokenController.java

```diff
@@ -4,18 +4,18 @@
 import org.springframework.http.HttpHeaders;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import ua.volosiuk.mytokenservice.dto.CredentialsDTO;
+import static ua.volosiuk.mytokenservice.util.CredentialsUtils.getCredentials;
 
-@Log4j2
 @RestController
+@RequestMapping("/token")
 public class TokenController {
 
-    @PostMapping("/token")
+    @PostMapping
     public String token(@RequestHeader(value = HttpHeaders.AUTHORIZATION) String headerContent) {
```

Comment:
"value =" part is redundant, pls check RequestHeader source code

Reply:
done